### ISSUE TITLE
Is p2p screen sharing still supported?

### ISSUE THREAD

**Issue by jusahah** [muaz-khan/Chrome-Extensions#88]
Hey,

first of all, great work with developing these extensions! 

Lately however, `desktopCapture-p2p` has stopped working when used over public internet. I guess this is an issue with STUN/TURN, as the extension works perfectly when used over localhost or local network.

### Issue

When using `desktopCapture-p2p` over public network, the connection is made (i.e extension shows that a viewer is present), but no video stream is shown in the viewer-side. On the viewer page, console is logged with messages like this:

```
adding candidate candidate:3511702143 1 udp 2113937151 {ip} 49421 typ host generation 0 ufrag uwwl network-cost 999
```
This happens with latest versions of Chrome and Firefox.

------------------------

Wondering if you have any tips or guide how to solve this?

Thanks.

**Comment by wyudong** [comment]
Have you got the solution?

**Comment by jusahah** [comment]
No, I haven't. If anyone knows how to solve this, lets share. From what I've gathered from Google searches, this seems to be common problem for many people. :)

**Comment by wyudong** [comment]
I installed [coturn](https://github.com/coturn/coturn) today to replace the STUN/TURN from google with my own STUN/TURN.

This [article](https://red5pro.com/docs/server/turnstun.html) may be helpful to install coturn. After coturn is installed, I have problem to check STUN/TURN in Chrome using [this tool](https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/), however, it's OK for Firefox. I guess it's not the server's fault but the Chrome's (also found [this issue](https://github.com/webrtc/samples/issues/1227)). After you test the connection successfully in Firefox, continue to replace the STUN/TURN urls in desktopCapture-p2p code. Don't forget to check the urls in these three files:
  1. AntMediaWrapper.js
  2. IceServersHandler.js
  3. RTCMultiConnection.min.js 

I also updated libs, RTCMultiConnection.min.js and socket.io.js to the latest version, but I think it's optional to solve the issue.
